examples/ransomware_detection/stages/create_features.py

```diff
@@ -82,88 +82,37 @@ def accepted_types(self) -> typing.Tuple:
     def supports_cpp_node(self):
         return False
 
-    def on_next(self, x: AppShieldMessageMeta):
 
-        snapshot_fea_dfs = []
+    def on_next(self, x: AppShieldMessageMeta):
 
         df = x.df
 
-        # Type cast CommitCharge.
-        df["CommitCharge"] = df["CommitCharge"].astype("float").astype("Int32")
         df["Name"] = df["Name"].str.lower()
 
         # Create PID_Process feature.
-        df['PID_Process'] = df.PID + '_' + df.Process
-
-        snapshot_ids = df.snapshot_id.unique()
-
-        if len(snapshot_ids) > 1:
-            # Group snapshot rows using snapshot id.
-            all_dfs = [df[df.snapshot_id == snapshot_id] for snapshot_id in snapshot_ids]
-        else:
-            all_dfs = [df]
-
-        extract_func = self._fe.extract_features
-        combine_func = FeatureExtractor.combine_features
+        df['PID_Process'] = df.PID.astype(str) + '_'# + df.Process
```

Comment:
Excess comment

Reply:
@elishahaim do we need concatenate `_ ` to `PID_Process `even though the process is being commented out?

Reply:
I need to verify it, if we really need the underscore

Reply:
@bsuryadevara I checked it, and we can remove it